packages/pds/tests/sync/subscribe-repos.test.ts

```diff
@@ -366,93 +330,93 @@ describe('repo subscribe repos', () => {
-      await sc.createAccount('baddie3.test', {
-        email: 'baddie3@test.com',
-        handle: 'baddie3.test',
-        password: 'baddie3-pass',
-      })
-    ).did
-
-    await randomPost(baddie3)
-    await sc.updateHandle(baddie3, 'baddie3-update.test')
-
-    await ctx.services.record(db).deleteForActor(baddie3)
-    await ctx.services.repo(db).deleteRepo(baddie3)
-    await ctx.services.account(db).deleteAccount(baddie3)
-
-    const ws = new WebSocket(
-      `ws://${serverHost}/xrpc/com.atproto.sync.subscribeRepos?cursor=${-1}`,
-    )
-
-    const gen = byFrame(ws)
-    const evts = await readTillCaughtUp(gen)
-    ws.terminate()
-
-    const didEvts = getAllEvents(baddie3, evts)
-    expect(didEvts.length).toBe(1)
-    verifyTombstoneEvent(didEvts[0], baddie3)
-  })
-
-  it('does not return invalidated events', async () => {
-    await sc.updateHandle(alice, 'alice3.test')
-    await sc.updateHandle(alice, 'alice4.test')
-    await sc.updateHandle(alice, 'alice5.test')
-    await sc.updateHandle(bob, 'bob3.test')
-    await sc.updateHandle(bob, 'bob4.test')
-    await sc.updateHandle(bob, 'bob5.test')
-
-    const ws = new WebSocket(
-      `ws://${serverHost}/xrpc/com.atproto.sync.subscribeRepos?cursor=${-1}`,
-    )
-
-    const gen = byFrame(ws)
-    const evts = await readTillCaughtUp(gen)
-    ws.terminate()
-
-    const handleEvts = getHandleEvts(evts)
-    expect(handleEvts.length).toBe(2)
-    verifyHandleEvent(handleEvts[0], alice, 'alice5.test')
-    verifyHandleEvent(handleEvts[1], bob, 'bob5.test')
-  })
-
-  it('sends info frame on out of date cursor', async () => {
-    // we rewrite the sequenceAt time for existing seqs to be past the backfill cutoff
-    // then we create some new posts
-    const overAnHourAgo = new Date(Date.now() - HOUR - MINUTE).toISOString()
-    await db.db
-      .updateTable('repo_seq')
-      .set({ sequencedAt: overAnHourAgo })
-      .execute()
-
-    await makePosts()
-
-    const ws = new WebSocket(
-      `ws://${serverHost}/xrpc/com.atproto.sync.subscribeRepos?cursor=${-1}`,
-    )
-    const [info, ...evts] = await readTillCaughtUp(byFrame(ws))
-    ws.terminate()
-
-    if (!(info instanceof MessageFrame)) {
-      throw new Error('Expected first frame to be a MessageFrame')
-    }
-    expect(info.header.t).toBe('#info')
-    const body = info.body as Record<string, unknown>
-    expect(body.name).toEqual('OutdatedCursor')
-    expect(evts.length).toBe(40)
-  })
-
-  it('errors on future cursor', async () => {
-    const ws = new WebSocket(
-      `ws://${serverHost}/xrpc/com.atproto.sync.subscribeRepos?cursor=${100000}`,
-    )
-    const frames = await readTillCaughtUp(byFrame(ws))
-    ws.terminate()
-    expect(frames.length).toBe(1)
-    if (!(frames[0] instanceof ErrorFrame)) {
-      throw new Error('Expected ErrorFrame')
-    }
-    expect(frames[0].body.error).toBe('FutureCursor')
-  })
+  // it('account deletions invalidate all seq ops', async () => {
```

Comment:
Just a note that this remains commented.  I think we want to ensure tombstones stick around now.